lib/RouteStore.js

```diff
@@ -23,6 +25,7 @@ var RouteStore = createStore({
         this.currentUrl = null;
         this.currentRoute = null;
         this.currentNavigate = null;
+        this.currentNavigateError = null;
```

Comment:
At the very least, `err.message` and `err.statusCode` should be sent. I agree with stack trace. Maybe we should be creating our own error object in the store instead of using the one that's passed in. This would help remove issues with client and server rendering differently.